src/OpenCV.h

```diff
@@ -22,7 +22,12 @@
 #include <opencv2/highgui.hpp>
 #include <opencv2/imgproc.hpp>
 #include <opencv2/videoio.hpp>
+#include <opencv2/opencv_modules.hpp>
 #endif
+#if ((CV_MAJOR_VERSION == 2) && (CV_MINOR_VERSION >=4) && (CV_SUBMINOR_VERSION>=4))
+#define HAVE_OPENCV_FACE
```

Comment:
In CV3, `HAVE_OPENCV_FACE` is only defined if the face module from opencv_contrib is installed.